src/pages/documentation/upgrade-platform-sdks/analytics-comparison.md

```diff
@@ -10,11 +10,7 @@ The Adobe Experience Platform Analytics extension uses [tags](https://experience
 2. Configure your mobile app with the create mobile property.<br/>The AEP Mobile Core extension provides general functionality required by all the Adobe AEP extensions. The Configuration extension is built into the Mobile Core and contains the configureWithAppId API. This API is used to link the tag mobile property with your mobile app. The documentation for this API can be seen at the [Configuration API Reference](../mobile-core/configuration/api-reference.md#configurewithappid) page. A code sample showing the usage of this API is provided below.
```

Comment:
I think we should revamp this tutorial for migrating from v4 SDK to AEP 3.x directly and remove it from the ACP repo

Reply:
We will come back to this once all updates have been made.